packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -74,8 +76,8 @@
 #include <memory>
 
 // need to interface this into cmake variable (or only use this flag when it is necessary)
```

Comment:
For the future, it would be good to wrap up the profiling API in an interface that hides how it's implemented.  That would minimize exposure of Ifpack2 code to macros and macrology.